Gemfile.lock

```diff
@@ -122,6 +122,8 @@ GEM
     factory_bot_rails (4.11.1)
       factory_bot (~> 4.11.1)
       railties (>= 3.0.0)
+    faker (1.9.1)
+      i18n (>= 0.7)
```

Comment:
Why is i18n added in this PR?

Reply:
after talking with the team we realised that this was something that came along with faker